.pre-commit-config.yaml

```diff
@@ -59,4 +59,4 @@ repos:
     rev: "1.7.0"
     hooks:
       - id: bandit
-        args: ["--exclude", ".venv,tests"]
+        args: ["--exclude", ".venv,tests", "--skip", "B108"]
```

Comment:
I would not turn on the overall sikipping of the B108 error, from what I understand it is linked to the failed security check "hardcoded temp dir" of line 24 of gunicorn.conf.py.
I would personally remove `--skip B108` from here, suggesting alternative workarounds:
- add a `# nosec` comment at the end of row 24 of `gunicorn.conf.py`
- set the temporary directory as environment variable `os.getenv("WORKER_TMP_DIR", tempfile.gettempdir())` without an hardcoded fallback, but using tempfile as suggested here https://security.openstack.org/guidelines/dg_using-temporary-files-securely.html